agent/llm.py

```diff
@@ -0,0 +1,14 @@
+def get_client(model_str):
+    provider, model = model_str.split(":")
```

Comment:
Here, you need to make sure model names with a ":" in it also works
Use https://github.com/phospho-app/phospho/blob/dev/phospho-python/phospho/lab/language_models.py#L29 as a reference

```
split_result = model.split(":")
if len(split_result) == 1:
    # Assume default provider to be openai
    provider = "openai"
    model_name = split_result[0]
elif len(split_result) > 2:
    # Some model names have :, so we need to join the rest of the string
    provider = split_result[0]
    model_name = ":".join(split_result[1:])
else:
    provider = split_result[0]
    model_name = split_result[1]
```

Reply:
Should be correct now